Documentation/gettingstarted/kubeproxy-free.rst

```diff
@@ -70,6 +70,13 @@ the token returned by ``kubeadm init``:
 
    kubeadm join <..>
 
+.. note::
+
+    Please ensure that
+    `kubelet <https://kubernetes.io/docs/reference/command-line-tools-reference/kubelet/>`_'s
+    ``--node-ip`` is set correctly on each worker if you have multiple interfaces.
+    Cilium's kube-proxy replacement may not work correctly otherwise.
```

Comment:
Maybe you could add something along these lines: "You can validate this by running `kubectl get nodes` to see whether each node has InternalIP which is assigned to a device with the same name on each node."